src/Sulu/Bundle/MediaBundle/Api/Media.php

```diff
@@ -668,7 +682,7 @@ public function getChanger()
      */
     public function getCreated()
     {
-        return $this->getFileVersion()->getCreated();
```

Comment:
its the special Media API object 
You dont wanted the created date of the FileVersion there you want the created date of the Media